src/Illuminate/Foundation/Console/OptimizeCommand.php

```diff
@@ -96,19 +100,22 @@ protected function compileClasses()
     }
 
     /**
-     * Get the node traverser used by the command.
+     * Get the class preloader used by the command.
      *
-     * @return \ClassPreloader\Parser\NodeTraverser
+     * @return \ClassPreloader\ClassPreloader
      */
-    protected function getTraverser()
```

Comment:
> we'd have only been calling it if the user had 2.x installed, which is breaking just the same as removing the method.

Nice arguments, as if saying we do support 2.x, but actually we don't. What is it so hard to keep a simple BC in Laravel. 5.2 is just within a month.